naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/impl/NamingKvStorage.java

```diff
@@ -97,7 +97,7 @@ public void put(byte[] key, byte[] value) throws KvStorageException {
             storage.put(key, value);
         } catch (Exception e) {
             throw new KvStorageException(ErrorCode.KVStorageWriteError.getCode(),
-                    "Put data failed, key: " + new String(key), e);
+                    "Put data failed, key: " + new String(key) + ",detail: " + e.getMessage(), e);
```

Comment:
There should be a space between `,` and `detail`.
Not only `put` operation, `get`, `delete` also need this enhancement.

Reply:
done.